package.json

```diff
@@ -37,6 +37,7 @@
     "lint:fix": "eslint . --ext js,json --fix",
     "lint:changed": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' --regexp='[.]json$' | tr '\\n' '\\0' | xargs -0 eslint",
     "lint:changed:fix": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' --regexp='[.]json$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
+    "lint:shellcheck": "shellcheck --version && find . -type f -name '*.sh' ! -path './node_modules/*' | xargs shellcheck",
```

Comment:
Shellcheck has this to say about that script: https://github.com/koalaman/shellcheck/wiki/SC2038